assets/src/edit-story/components/dropDown/index.js

```diff
@@ -0,0 +1,320 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import React, { useState, useEffect, useCallback } from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import { rgba } from 'polished';
+import onClickOutside from 'react-onclickoutside';
+import { debounce } from 'throttle-debounce';
+
+/**
```

Comment:
I think that you don't need this `WordPress dependencies` declaration. Thoughts?

Reply:
Now using __ for translation 😄